scripts/bin/common

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+: ${WATCH_FILE:=/tmp/mounts.fifo}; readonly WATCH_FILE
+
+mklog() {
+    _log_dir="${1:-"${LOG_DIR:="$(mktemp -d)"}"}"
+
+    [ -d "$_log_dir" ] || mkdir -p "$_log_dir"
+
+    echo "${_log_dir}/$(date -Iseconds).log"
```

Comment:
`echo` -> `touch`?

Reply:
As it's currently used, `mklog()` is called just to generate an appropriate name, and is implicitly created when `log()` is called. That beings said, I do think it's probably worth initializing empty logs, though. An empty log is definitely better than no log.

Reply:
As it's currently used, mklog() is called just to generate an appropriate name, and is implicitly created when log() is called. That beings said, I do think it's probably worth initializing empty logs, though. An empty log is definitely better than no log.

Reply:
I think it would be good to create the empty log because what happens if we can't actually write to the filesystem (imagine an SD card with write-lock on).